index.html

```diff
@@ -357,6 +375,9 @@ <h3>Weighing increased fingerprinting surface</h3>
             <p class="practicedesc">
               What browsing contexts, resources and requests need access to a particular feature?  Identifiers can often be scoped to have a different value in different origins. Some configuration may only be necessary in top-level browsing contexts.
             </p>
+            <p class="practicedesc">
+              If an event is to be fired in response to an environmental or hardware change, can that event be fired only on visible documents, and queued for firing in non-visible documents? Should it be restricted by an iframe sandbox?
```

Comment:
Spec authors do this more reliably if we give them specific terms to hook into:

```suggestion
              If an event is to be fired in response to an environmental or hardware change,
              only fire that event when the {{Window}}'s [=associated document=]'s [=Document/visibility state=] is "`visible`",
              or in {{Worker}}s whose [=WorkerGlobalScope/owner set=] includes such a {{Document}}.
              If background pages need to learn of the event when they're focused,
              also fire the event while <a data-cite="HTML#update-the-visibility-state">updating the visibility state</a>.
              Consider whether it should be restricted by an iframe sandbox.
```

In theory, fuzzing the timing of the event can also help, but I think the WG has settled on just visible-only being enough.

Reply:
Thanks, updated!